static/js/recent_topics.js

```diff
@@ -317,6 +319,18 @@ function format_topic(topic_data) {
     const senders = all_senders.slice(-MAX_AVATAR);
     const senders_info = people.sender_info_for_recent_topics_row(senders);
 
+    const extra_sender_ids = all_senders.slice(0, -MAX_AVATAR);
+    const displayed_other_senders = extra_sender_ids.slice(-MAX_EXTRA_SENDERS);
+    let other_senders_fullname_string = people
+        .get_display_full_names(displayed_other_senders.reverse())
+        .join("<br>");
+    if (extra_sender_ids.length > MAX_EXTRA_SENDERS) {
+        const extra_sender_count = extra_sender_ids.length - MAX_EXTRA_SENDERS;
+        other_senders_fullname_string = other_senders_fullname_string.concat(
+            $t({defaultMessage: `<br/>and {extra_sender_count} others.`}, {extra_sender_count}),
```

Comment:
The `<br/>` should be kept outside of the translateable string, and ideally just added via the `join` operation you have above.

With a bit of refactoring to collect a list (and have this block just append an extra item) and then join with `<br />` items at the end, this'll read a lot more cleanly.

Reply:
Updated this.